tests/test_models.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+
+from django.core.urlresolvers import reverse
+from . import factories
+
+
+class TestAgent:
+
+    def test_unicode(self):
+        agent = factories.AgentFactory.build()
+        assert agent.agent_name == unicode(agent)
+
+    def test_get_absolute_url(self):
+        agent = factories.AgentFactory.build()
+        url = reverse(
```

Comment:
This line is basically the exact code of the method this test is testing. Should we handle this test differently?